generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.html

```diff
@@ -36,7 +36,7 @@ <h4 class="modal-title" id="my<%= entityClass %>Label" jhi-translate="<%= keyPre
     <div class="form-group"<% if ((fieldType == 'byte[]' || fieldType === 'ByteBuffer') && fieldTypeBlobContent != 'text') { %> ngf-drop ngf-change="set<%=fieldNameCapitalized %>($file, <%= entityInstance %>)"<% if (fieldTypeBlobContent == 'image') { %> ngf-pattern="'image/*'"<% } %><% } %>>
     <label class="form-control-label" jhi-translate="<%= translationKey %>" for="field_<%= fieldName %>"><%= fieldNameHumanized %></label>
     <%_ if (fields[idx].fieldIsEnum) { _%>
-    <select class="form-control" name="<%= fieldName %>" [(ngModel)]="<%= entityInstance %>.<%= fieldName %>" id="field_<%= fieldName %>" #<%= fieldName %>="ngModel"<% if (fields[idx].fieldValidate == true && fields[idx].fieldValidateRules.indexOf('required') != -1) { %> required<% } %>>
```

Comment:
Was the objective of removing these variable names from the controls to access the same controls through the form object in validation messages?